### DISCUSSION TITLE
milvus集群创建collection成功但proxy日志提示“send replicate msg failed”

### DISCUSSION THREAD

**Discussion by walker1024** [milvus-io/milvus#38122] [Q&A and General discussion] [answered]
milvus集群创建collection，可以成功创建，但是查看proxy日志提示“send replicate msg failed”，集群可以正常读写(向量可正常写入和检索)。
### 集群配置
proxy：1
mixcoord：1
indexnode：2
datanode：1
querynode：1
kafka：外部集群
etcd：外部集群
milvus版本：2.4.11

### 异常日志

> [2024/11/30 14:27:47.808 +00:00] [WARN] [proxy/util.go:1736] ["send replicate msg failed"] [pack="{\"BeginTs\":454289488023126024,\"EndTs\":454289488023126024,\"Msgs\":[{\"Ctx\":{\"Context\":{\"Context\":{\"Context\":{\"Context\":{\"Context\":{}}}}}},\"BeginTimestamp\":454289488023126024,\"EndTimestamp\":454289488023126024,\"HashValues\":[0],\"MsgPosition\":null,\"base\":{\"msg_type\":106,\"msgID\":454289488023126024,\"timestamp\":454289488023126024,\"sourceID\":92},\"db_name\":\"default\",\"collection_name\":\"stress_test\"}],\"StartPositions\":null,\"EndPositions\":null}"] [error="Local: Unknown topic"]
[2024/11/30 14:27:47.808 Z] [GIN] [/v2/vectordb/collections/create] [traceID=6c133cbaa5fbaf50f465ffadc4340c9e] [code=200] [latency=35.283298ms] [client=*.*.*.*] [method=POST] [error=]

请教各位大佬，这种一般是什么问题？

`kafka集群亲测可以正常读写`

**Comment by yhmo** [comment, marked as answer]
找不到topic，可能是没有创建kafka topic的权限。milvus启动时需要创建若干个topic

**Reply by walker1024** [reply]
问题已解决，查了kafka集群配置及已生成的topic列表，是kafka集群同步问题，已解决，感谢 @yhmo

**Comment by lovmatong2** [comment]
你好，你是用kubeadm安装的多节点k8s吗，milvus怎么在其之上实现部署呢，我现在正在面临这个问题，官方给的minikube是单节点集群部署不适合我要做的业务，假若您方便，请帮助我一下。

**Reply by xiaofan-luan** [reply]
使用helm安装是最方便的
https://milvus.io/docs/upgrade_milvus_standalone-helm.md#Upgrade-Milvus-Standalone-with-Helm-Chart

**Reply by walker1024** [reply]
使用helm部署比较简单：
https://milvus.io/docs/upgrade_milvus_standalone-helm.md#Upgrade-Milvus-Standalone-with-Helm-Chart
我目前使用的就是这种方式

**Reply by xiaofan-luan** [reply]
可以联系小龙，我们有独立的镜像源
